Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM openjdk:8-alpine
 
 # build and cleanup
 COPY setup.sh /opt/setup.sh
-RUN chmod +x /opt/setup.sh && /opt/setup.sh
```

Comment:
Running the 2 commands chained together gave some issues on linux, let me see whether I can reproduce that.